contracts/utils/ICS26AdminsDeprecated.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.28;
+
+/// @title ICS26AdminsDeprecated
+/// @notice This contract is a library for removing the deprecated and deleted IBCUUPSUpgradeable contract.
+/// @dev This library will be deleted in the next release.
+library ICS26AdminsDeprecated {
+    /// @notice Storage of the IBCUUPSUpgradeable contract
+    /// @dev It's implemented on a custom ERC-7201 namespace to reduce the risk of storage collisions when using with
+    /// upgradeable contracts.
+    /// @param timelockedAdmin The timelocked admin address, assumed to be timelocked
+    /// @param govAdmin The governance admin address
+    struct IBCUUPSUpgradeableStorage {
+        address timelockedAdmin;
+        address govAdmin;
+    }
+
+    /// @notice ERC-7201 slot for the IBCUUPSUpgradeable storage
+    /// @dev keccak256(abi.encode(uint256(keccak256("ibc.storage.IBCUUPSUpgradeable")) - 1)) & ~bytes32(uint256(0xff))
+    bytes32 private constant IBCUUPSUPGRADEABLE_STORAGE_SLOT =
```

Comment:
Is there anything we need to do with this storage slot after we're "done" with it? Or do we just delete this and pretend it didn't exist?

Reply:
delete and pretend it didn't exist. But I do actually wipe the slots as well